### ISSUE TITLE
Make date of birth inputs consistent across ARP

### ISSUE THREAD

**Issue by darbyemadewell** [department-of-veterans-affairs/va.gov-team#138654]
## Background/Context

On most ARP pages, the Date of Birth input consists of a Month select with Day and Year text inputs. On Find Claimant, the Date of Birth input consists of a Month select and a Day select with only a Year text input.

## Expected vs. Actual
**Expected**: 
The Date of Birth inputs should be consistent across ARP and use the Month select, Day text input, Year text input format.

**Actual**: 
The Date of Birth inputs are not consistent across ARP.
